packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -987,6 +987,7 @@
   "Mentions": "Mentions",
   "Mentions_default": "Mentions (default)",
   "Message": "Message",
+  "Message_AllowAttachTooLongMessages": "Allow attach too long messages",
```

Comment:
@geekgonecrazy any suggestion here?

Reply:
`Convert long messages to attachment` would be a bit better

Reply:
Yep... This message appear for admins so that they allow/deny the converting of long messages to attachment. It sounds good: "Message_AllowConvertLongMessagesToAttachment"?